service/pom.xml

```diff
@@ -82,6 +82,14 @@
       <artifactId>slf4j-api</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>io.camunda</groupId>
+      <artifactId>zeebe-gateway</artifactId>
+      <classifier>tests</classifier>
+      <type>test-jar</type>
+      <scope>test</scope>
+    </dependency>
+
```

Comment:
❓ I'm wondering what has caused this dependency addition,

Reply:
That's for `StubbedBrokerClient`

Reply:
Ah I see, nice addition :)